util/config/fixpath.py

```diff
@@ -25,7 +30,7 @@ def main(env='PATH', delim=':'):
     pattern = r'(?<!\\)\:'
```

Comment:
This change may be a simpler way to accomplish the same thing:

```python
 pattern = r'(?<!\\)\{0}'.format(delim)
```

Reply:
Ah, I am sorry I misunderstood the original issue (not a fish user, as you can probably tell)